Scripts/utils.py

```diff
@@ -5,6 +5,7 @@
 import Scripts.auth as auth
 from googleapiclient.errors import HttpError
 from html import unescape
+import shutil
```

Comment:
this code is ONE LINE

Reply:
Would you suggest adding more lines to fix the issue?

Reply:
> Would you suggest adding more lines to fix the issue?

 I tried this myself, and it does fix the issue. However, if you have other ideas or improvements, I'm happy to discuss them.